control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -115,6 +115,7 @@ impl From<VolumeSpec> for volume::VolumeDefinition {
                 topology: volume_spec.topology.map(|topology| topology.into()),
                 last_nexus_id: nexus_id.map(|id| id.to_string()),
                 thin: volume_spec.thin,
+                volume_group: volume_spec.volume_group.map(|vg| vg.into()),
```

Comment:
nit: might be able to use into_opt, but both are fine.

Reply:
Done